### PR TITLE
pull_40

### PR DESCRIPTION
**Pull request by loctx-2273**

### CODE REVIEW

**Comment by loctx-2273** [comment]
@hungnv-1238 fff

**Comment by loctx-2273** [comment]
@hungnv-1238 fff

**Comment by loctx-2273** [comment]
payload["comment"].present? && payload["comment"]["pull_request_review_id"].present?

**Comment by loctx-2273** [comment]
@hungnv-1238 fff

**Comment by loctx-2273** [comment]
@hungnv-1238 fff

**Comment by tranloc171197** [comment]
sdfsdfsdf

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ def set_locale
   end
 
   def default_url_options
-    {locale: I18n.locale}
+    # {locale: I18n.locale}
```
Comment (tranloc171197) [inline app/controllers/application_controller.rb:12]: sdàfsd @loctx-2273 @tranloc171197

**Comment by tranloc171197** [comment]
@taipa-2180 Test

**Comment by tranloc171197** [comment]
@taipa-2180 test

**Comment by loctx-2273** [comment]
@taipa-2180 test

**Comment by tranloc171197** [comment]
LGTM

**Pull request merged**